builtin/logical/transit/backend.go

```diff
@@ -122,11 +122,13 @@ func (b *backend) GetPolicy(ctx context.Context, polReq keysutil.PolicyRequest,
 		currentCacheSize := b.lm.GetCacheSize()
 		storedCacheSize, err := GetCacheSizeFromStorage(ctx, polReq.Storage)
 		if err != nil {
+			b.configMutex.RUnlock()
```

Comment:
These code changes are 🆗, but please consider making the equivalent changes as you did in the Transform plugin (extracting a method, changing the if statement into a guard).

Reply:
I'd like to get this merged ASAP so as not to block the point releases.  Divya: can you follow up sometime after with a PR that aligns this to the transform changes?

Reply:
Sure. I will open a new PR for new helper method.